examples/deadline/All-In-AWS-Infrastructure-Basic/python/package/config.py

```diff
@@ -46,6 +46,7 @@ def __init__(self):
         self.deadline_client_linux_ami_map: Mapping[str, str] = {'us-west-2': 'ami-04ae356533dc07fb5'}
 
         # A secret (in binary form) in SecretsManager that stores the UBL certificates in a .zip file.
+        # This must be a complete ARN, including the "-<random text>" suffix.
```

Comment:
including the random text a little bit confusing as for me.
```suggestion
        # This must be in format `arn:aws:secretsmanager:<Region>:<AccountId>:secret:<SecretName>-<6RandomCharacters>`
```

Reply:
Good suggestion! I'll make that change.